tx/textual/internal/testdata/ledger.json

```diff
@@ -0,0 +1,91 @@
+		"expected": [
+			"0 | Chain id: my-chain",
+			"1 | Account number: 1",
+			"2 | Sequence: 2",
+			"3 | Transaction: 1 Messages",
+			"4 | > Message (1/1): cosmos.bank.v1beta1.MsgSend",
+			"5 | > From address: cosmos102hty0jv2s29lyc4u0tv97z9v298e24t3vwtpl",
+			"6 | > To address: cosmos102hty0jv2s29lyc4u0tv97z9v298e24t3vwtpl",
+			"7 | > Amount: 10 ATOM",
+			"8 | End of Messages",
+			"9 | Fee: 0.002 ATOM"
+		],
+		"expert": false
+	},
+	{
+		"name": "minimal in expert mode",
+		"tx": "90a10172436861696e2069643a206d792d636861696ea101714163636f756e74206e756d6265723a2031a1016b53657175656e63653a2032a201782a5075626c6963206b65793a20636f736d6f732e63727970746f2e736563703235366b312e5075624b657903f5a3016d5075624b6579206f626a656374020103f5a30178314b65793a2041757664662b54393633626369694265396c3135444e4d4f696a64615843556f367a71534f76483754586c4e020203f5a101775472616e73616374696f6e3a2031204d65737361676573a201782a4d6573736167652028312f31293a20636f736d6f732e62616e6b2e763162657461312e4d736753656e640201a201783b46726f6d20616464726573733a20636f736d6f7331756c6176336873656e7570737771666b77327933737570356b677471776e76716138657968730201a2017839546f20616464726573733a20636f736d6f7331656a726634637572327779366b667572673966326a707070326833616665356836706b6835740201a2016f416d6f756e743a2031302041544f4d0201a1016f456e64206f66204d65737361676573a10178184d656d6f3a20e29a9befb88fe29a9befb88fe29a9befb88fa10171466565733a20302e303032207561746f6da20172476173206c696d69743a203130302730303003f5a201785348617368206f66207261772062797465733a206532333764633265336638663062316430333130653163643062306162356663346135396534626333343534643637656236356333666462306661353939633903f5",
+		"expected": [
+			"0 | Chain id: my-chain",
+			"1 | Account number: 1",
+			"2 | Sequence: 2",
+			"3 | Public key: cosmos.crypto.secp256k1.PubKey",
+			"4 | > PubKey object",
+			"5 | >> Key: Auvdf+T963bciiBe9l15DNMOijdaXCUo6zqSOvH7TXlN",
+			"6 | Transaction: 1 Messages",
+			"7 | > Message (1/1): cosmos.bank.v1beta1.MsgSend",
+			"8 | > From address: cosmos1ulav3hsenupswqfkw2y3sup5kgtqwnvqa8eyhs",
+			"9 | > To address: cosmos1ejrf4cur2wy6kfurg9f2jppp2h3afe5h6pkh5t",
+			"10 | > Amount: 10 ATOM",
+			"11 | End of Messages",
+			"12 | Fee: 0.002 ATOM",
+			"13 | Gas limit: 100'000",
+			"14 | [1/2] Hash of raw bytes: e237dc2e3f8f0b1d0310e1cd0b0ab5fc4a59",
+			"15 | [2/2] e4bc3454d67eb65c3fdb0fa599c9"
```

Comment:
I wasn't sure where exactly to cut into 2 lines, lmk where you cut in general, and I can update the test cases